pkg/tso/keyspace_group_manager.go

```diff
@@ -728,6 +728,9 @@ func (kgm *KeyspaceGroupManager) GetElectionMember(
 	if err != nil {
 		return nil, err
 	}
+	if am == nil {
```

Comment:
When will this happen?

Reply:
check if it is primary after `kgm.deleteKeyspaceGroup` with default keyspace group

Reply:
Do you mean delete the default keyspace group?

Reply:
the default keyspace group no longer contain this member

Reply:
If it actually goes here, it will panic I think.

Reply:
it can be removed after @binshi-bing has fixed